botocore/httpchecksum.py

```diff
@@ -462,12 +469,16 @@ def _handle_bytes_response(http_response, response, algorithm):
     "sha1": Sha1Checksum,
     "sha256": Sha256Checksum,
 }
+_CRT_CHECKSUM_CLS = {}
 
 if HAS_CRT:
     # Use CRT checksum implementations if available
-    _CHECKSUM_CLS.update(
-        {"crc32": CrtCrc32Checksum, "crc32c": CrtCrc32cChecksum}
-    )
+    _CRT_CHECKSUM_CLS = {
+        "crc32": CrtCrc32Checksum,
+        "crc32c": CrtCrc32cChecksum,
+    }
```

Comment:
Saving this to a variable instead lets us validate the keys match the hardcoded list when the CRT is present.

We can't take the list of keys when the CRT is present because it will be empty and is needed for the custom exception.